src/ruleset_loading.jl

```diff
@@ -139,3 +140,5 @@ function _safe_hook_fun(hook_fun, sig)
         @error "Error triggering hook" hook_fun sig exception=(err, catch_backtrace())
     end
 end
+
+precompile(_package_hook, (Base.PkgId,))
```

Comment:
Can we put this in a seperate file?
`precompile.jl`

We should probably be precompiling a bunch of things

Reply:
Ok, will do